lib/TopStoriesFeed.jsm

```diff
@@ -705,10 +705,8 @@ this.TopStoriesFeed = class TopStoriesFeed {
       return;
     }
     switch (action.type) {
-      // Check for pref initial values to lazy load activity stream top stories
-      // Here we are not using usual INIT and relying on PREFS_INITIAL_VALUES
-      // to check discoverystream pref and load activity stream top stories only if needed.
-      case at.PREFS_INITIAL_VALUES:
+      // Check discoverystream pref and load activity stream top stories only if needed.
+      case at.INIT:
         this.lazyLoadTopStories(action.data[DISCOVERY_STREAM_PREF]);
```

Comment:
We could probably make lazyLoadTopStories use `getState().Prefs` if we don't pass in a value here or above handleDisabled.